app/views/granules/download.html.erb

```diff
@@ -1,19 +1,90 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <title><%= site_page_title('Downloads') %></title>
-    <link rel="shortcut icon" href="/favicon.ico?v=1.1">
-  </head>
-  <body>
-    <ul>
-      <% @urls.each do |url| %>
-      <li><a href="<%= url %>"><%= url %></a></li>
-      <% end %>
-    </ul>
-    <% if  @errors.present? %>
-      <%= @errors %>
-     <% end %>
-  </body>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title><%= site_page_title('Downloads') %></title>
+  <link rel="shortcut icon" href="/favicon.ico?v=1.1">
+</head>
+<body>
+<div id="errors"></div>
+<div id="loading">Loading more...</div>
+<ul id="links">
+</ul>
+
+<script>
+  (function () {
+
+    var browseOnly = <%= @query.delete('browse') %>;
+    var currentPage = <%= @query['page_num'] %>;
+    var project = <%= @query['project'] %>;
+    var collection = '<%= @query['collection'] %>';
+    var cmrHits = -1;
+
+    function getNextPageUrl() {
+      queryPageNum = parseInt(url.match(/page_num=(\d+)/)[1]);
```

Comment:
I think CMR may be returning a link rel="next" element now. Check and use that if so.